pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -57,17 +57,21 @@ def __init__(
                 f" Move your implementation to `{self.__class__.__name__}.teardown()` instead."
             )
 
-    @property
-    def checkpoint_io(self) -> CheckpointIO:
-        return self._checkpoint_io
-
     @property
     def precision_plugin(self) -> PrecisionPlugin:
         return self._precision_plugin
 
+    @precision_plugin.setter
+    def precision_plugin(self, plugin: PrecisionPlugin) -> None:
+        self._precision_plugin = plugin
```

Comment:
This has been added for consistency. Ideally neither would have a setter but the `AcceleratorConnector` requires it.

Reply:
How is this not creating an error? Something is not right here

Reply:
Why would it?